xclim/core/options.py

```diff
@@ -141,44 +141,43 @@ def run_check(*args, **kwargs):
 class set_options:
     """Set options for xclim in a controlled context.
 
-    Currently-supported options:
-
-    - ``metadata_locales``:
-      List of IETF language tags or tuples of language tags and a translation dict, or
-      tuples of language tags and a path to a json file defining translation of attributes.
-      Default: ``[]``.
-    - ``data_validation``:
-      Whether to 'log',  'raise' an error or 'warn' the user on inputs that fail the data checks in `xclim.core.datachecks`.
-      Default: ``'raise'``.
-    - ``cf_compliance``:
-      Whether to 'log',  'raise' an error or 'warn' the user on inputs that fail the CF compliance checks in `xclim.core.cfchecks`.
-      Default: ``'warn'``.
-    - ``check_missing``:
-      How to check for missing data and flag computed indicators.
-      Default available methods are "any", "wmo", "pct", "at_least_n" and "skip".
-      Missing method can be registered through the `xclim.core.options.register_missing_method` decorator.
-      Default: ``'any'``
-    - ``missing_options``:
-      Dictionary of options to pass to the missing method. Keys must the name of
-      missing method and values must be mappings from option names to values.
-    - ``run_length_ufunc``:
+    Attributes
+    ----------
```

Comment:
Attributes? Does this trigger something in the doc formatting?

If no, I would suggest something more precise like:
```suggestion
    Options
    -------
```

Reply:
Allegedly, this is how we're supposed to document classes: https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring